Documentation/Major/PreupgradeTasks/ReferenceIndex.rst.txt

```diff
@@ -11,11 +11,17 @@ With command line (recommended)
    As the reference index might take quite long, especially on instances not
    running it regularly, an update via CLI is recommended.
 
-In a composer installation execute::
+In a composer installation execute:
+
+.. code-block:: shell
+   :caption: typo3_root$
```

Comment:
Hm, what does this mean exactly? A shell prompt? I'd rather use a textual caption instead here since this would cause more confusion than clarification.

Reply:
I want to clarify from where this command should be called, the directory you need to be in

Reply:
In this case it would be the directory where `vendor/bin` exists, which doesn't necessarily have to be the "typo3_root". Which by itself can be many:

- Composer root
- App root (can be different)
- TYPO3 root (one would expect `typo3/` here normally)
- Public root (e.g. `web/` or `public/`)

So IMO this is too ambiguous so we should skip it altogether.

Reply:
Removed it